version_control.go

```diff
@@ -307,12 +317,90 @@ func (c *VersionCache) UpdateBinary(ctx context.Context, binary string) (bool, e
 	return needRestart, c.save()
 }
 
-// returns true if mtype is any of expected strings.
-func mimeIsAny(mtype *mimetype.MIME, expected []string) bool {
-	for _, expectedType := range expected {
-		if mtype.Is(expectedType) {
-			return true
+func (c *VersionCache) CleanCache(ctx context.Context) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	// this can be set to the number of days to keep, ex: "VIAM_AGENT_FORCE_CLEAN=5"
+	forceVal := os.Getenv("VIAM_AGENT_FORCE_CLEAN")
```

Comment:
optional: put in agentOpts here https://github.com/viamrobotics/agent/blob/main/cmd/viam-agent/main.go#L36, will get docstring, default + int conversion for free (I assume -- have never used our parser library for ints)

Reply:
Didn't do that as this is almost certainly ONLY ever for testing. Secondly, I'd have to then pass the opts struct all the way down the stack to here. That didn't seem ideal for a "test-only" thing.